spec/system/authentication/user_logs_in_with_email_spec.rb

```diff
@@ -80,6 +80,16 @@
 
         expect(page).to have_current_path("/?signin=true")
       end
+
+      it "logs in and redirects to onboarding if it hasn't been seen" do
+        user.update(saw_onboarding: false)
+
+        visit sign_up_path
+        log_in_user(user)
+
+        expect(page).to have_current_path("/onboarding", ignore_query: true)
+        expect(page.html).to include("onboarding-container")
+      end
```

Comment:
Added this test to the email login specs to match what we do in the other system authentication specs like [this one](https://github.com/forem/forem/blob/master/spec/system/authentication/user_logs_in_with_facebook_spec.rb#L17-L23)